octoprint_mrbeam/analytics/usage_handler.py

```diff
@@ -39,11 +43,20 @@ def __init__(self, plugin):
 		self._subscribe()
 		self.log_usage()
 
+		self._laserheadHandler = laserheadHandler(plugin)
+
+		# Read laser head. If it's None, use 'no_serial'
+		self._lh = self._laserheadHandler.get_current_used_lh_data()
+		if self._lh['serial']:
+			self._laser_head_serial = self._lh['serial']
+		else:
+			self._laser_head_serial = 'no_serial'
+
+		self._init_missing_usage_data()
+
 	def log_usage(self):
-		self._logger.info("Usage: total: {}, current laser head: {} - {}".format( \
-			self._get_duration_humanreadable(self._usage_data['total']['job_time']), \
-			# self._get_duration_humanreadable(self._usage_data['air_filter']['job_time']), \
-			self._get_duration_humanreadable(self._usage_data['laser_heads'][-1]['job_time']), \
+		self._logger.info("Usage: total: {} - {}".format(
+			self._get_duration_humanreadable(self._usage_data['total']['job_time']),
```

Comment:
Would be good to have all usage data in the log. overall, air filter (pre & main), laserhead and gantry. Is this the case?

Reply:
Included